satellite/satellitedb/orders.go

```diff
@@ -49,6 +50,9 @@ func (db *ordersDB) UseSerialNumber(ctx context.Context, serialNumber storj.Seri
 	if err != nil {
 		return nil, err
 	}
+	if dbxSerialNumber == nil {
+		return nil, errs.New("serial number not found")
```

Comment:
Declare an error class `ErrUsingSerialNumber` in orders package, then in database code `ErrUsingSerialNumber.New("serial number not found")` or `ErrUsingSerialNumber.New("serial number already used")`. This will also make the endpoint error handling nicer.